ops/provision/modules/k8s/main.tf

```diff
@@ -0,0 +1,90 @@
+terraform {
+  required_providers {
+    rancher2 = {
+      source = "rancher/rancher2"
+      version = "1.11.0"
+    }
+    kubectl = {
+      source  = "gavinbunney/kubectl"
+      version = ">= 1.7.0"
+    }
+  }
+}
+
+
+provider "helm" {
+  kubernetes {
+    config_path = var.kubeconfig
+  }
+}
+
+provider "kubectl" {
+  config_path = var.kubeconfig
+}
+
+provider "kubernetes" {
+  config_path = var.kubeconfig
+}
+
+resource "helm_release" "ingress-nginx" {
+  name = "ingress-nginx"
+  namespace = "ingress-nginx"
+  create_namespace = true
+  version = "4.3.0"
+  repository = "https://kubernetes.github.io/ingress-nginx"
+  chart = "ingress-nginx"
+  values = [
+    file("modules/k8s/files/ingress-nginx-values.yaml")
+  ]
+}
+
```

Comment:
👇 i'm unsure what role this resource plays. Also not seeing a namespace for "storage",  so maybe this hasn't been applied, or maybe i'm missing it?

Reply:
this is the actual "hook up efs to the eks cluster" code. I see it in the cluster... not sure why you do not.

Reply:
I found it under "Coordination > Leases" in the Rancher menu, but still wondering what it is used for

Reply:
👍